.github/workflows/node.js.yml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+
+    - uses: actions/checkout@v2
+    - name: Setup
+      uses: actions/setup-node@v1
+
+    - name: Install
+      run: sudo apt-get install -y libunbound-dev |
+           npm install nyc coveralls
+
+    - name: Lint
+      run: npm run lint
+
+    - name: Test
+      run: node_modules/.bin/nyc --reporter lcov node_modules/.bin/bmocha -- --reporter spec test/*.js
```

Comment:
Maybe this code coverage command could be turned into an npm script so that it can also be ran locally?

Reply:
So, the thinking was that this is purely CI stuff.  Another reason is that we don't really have a pure-dep fork of `nyc` or `istanbul`. Adding those to devDeps would add a ton of unaudited packages to the dependency graph.

Reply:
Previously, `eslint` wasn't in the dev dependencies but a script existed that referenced it. I'm not super opinionated on needing this, I just think having the ability to look into code coverage locally is a nice to have.

Reply:
OK, restored the `test-ci` script.